extensions/amp-analytics/0.1/vendors/bluetriangle.json

```diff
@@ -0,0 +1,44 @@
+{
+    "vars": {
+        "siteID": "",
+        "pageName": "",
+        "trafficSegment": "",
+        "pageType": "AMP Default",
+        "guid": "$SUBSTR(${random},0,19)",
+        "sessionID": "$SUBSTR(${random},0,19)"
+    },
+    "requests": {
+        "host": "https://d.btttag.com",
+        "endpoint": "${host}/btt.gif?",
+        "pageview": "${endpoint}?gID=${guid}&sID=${sessionID}&siteID=${siteID}&pageName=${pageName}&txnName=${trafficSegment}&thisURL=${canonicalUrl}&RefURL=${documentReferrer}&dom=${navTiming(domLoading,domComplete)}&domContentLoadedEnd=${navTiming(domContentLoadedEventEnd)}&domContentLoadedStart=${navTiming(domContentLoadedEventStart)}&domInteractive=${navTiming(domInteractive)}&domLoading=${navTiming(domLoading)}&domainLookupStart=${navTiming(domainLookupStart)}&elementCount=0&fByte=${navTiming(requestStart,responseStart)}&fCPnt=${firstContentfulPaint}&fetchStart=${navTiming(fetchStart)}&firstPaint=0&fmp=0&fps=0&fps_tti=0&loadEventStart=${navTiming(loadEventStart)}&lt_tti=0&nst=${navTiming(navigationStart)}&orderTND=0&pSz=0&pWSz=0&pageValue=&pgTm=${navTiming(navigationStart, loadEventStart)}&redirectCount=${navRedirectCount}&redirectStart=${navTiming(redirectStart)}&requestStart=${navTiming(requestStart)}&responseStart=${navTiming(responseStart)}&secureConnectionStart=${navTiming(secureConnectionStart)}&ssl=${navTiming(secureConnectionStart, responseStart)}&tcp=${navTiming(connectStart, connectEnd)}&top=0&totRD=0&tti=0&tul=0&unloadEventStart=${navTiming(unloadEventStart)}&largestContentfulPaint=${largestContentfulPaint}&cumulativeLayoutShift=${cumulativeLayoutShift}&firstInputDelay=${firstInputDelay}&wcd=0&eT=1&navigationType=1"
+    },
+    "transport": {
+        "beacon": false,
+        "xhrpost": false,
+        "image": true
```

Comment:
AMP analytics defaults to beacon then xhrpost then image, if the previous transport methods are not supported. I want to confirm your server only supports image requests.

Reply:
Yes, we only support image requests.